scripts/jobs/stream-to-nfs-job.sh

```diff
@@ -0,0 +1,29 @@
+#! /bin/bash
```

Comment:
Remove this script. The existing script stream-to-file.job.sh can be used.

Reply:
removed